addons/addon-base-workflow/packages/base-workflow-core/lib/workflow/workflow-template-draft-service.js

```diff
@@ -139,6 +139,9 @@ class WorkflowTemplateDraftService extends Service {
     // Check if the owner of this draft is the same entity that is trying to update the draft
     if (originalDraft.uid !== by) throw this.boom.forbidden('You are not authorized to perform this operation', true);
 
+    if (!_.isObject(template)) throw this.boom.badRequest('The provided template is not an object', true);
```

Comment:
Should we use the word `json` instead of `object` since this error would be sent back to clients?